README.md

```diff
@@ -1 +1,193 @@
+- [Website description](https://www.loom.com/share/9f0775bb54844cac8a99654a34a77a94)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need an IDE such as VS code, sublime, atom e.t.c:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+ $ cd my-folder
+ $ git clone git@github.com:mutugi-c/GIS-DAY.git
+```
+
+### Install
+
+Install git to work on this project with:
+
+```sh
+   $ git config --global user.name "First Last"
+
+   $ git config --global user.email "example@example.com"
+```
+
+### Usage
+
+To run the project have Node.js installed, execute the following command:
+
+```sh
+  $ npm install http-server -g
+  $ http-server
+```
+
+Go to your browser and type localhost:8080. Your Application should run
+
+### Run tests
+
+To run tests, run the HTML document in Google Chrome and right-click the HTML element you want to inspect. Click on "Inspect" and you will have the tools to run, analyze, and even debug the code
+
+### Deployment
+
+You can deploy this project using build tools such as webpack, Rollup, Parcel, e.t.c or even deploy using Github pages:
+
+Example: To build for production with parcel, use the folowing commands:
+
+- Initialise npm package
+- Install parcel
+- Build for production
+
+```sh
+ $ npm init -y
+ $ npm i -g parcel-bundler
+ $ parcel build index.html
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Chris Mutugi**
+
+- GitHub: [@mutugi-c](https://github.com/mutugi-c)
+- Twitter: [@chris\_\_mutugi](https://twitter.com/chris__mutugi)
+- LinkedIn: [chris-mutugi](https://www.linkedin.com/in/chris-mutugi-46594b123/)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Future features that will be added to the project include:
+
+- **Links to any additional activities**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, give a ⭐️.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank [Cindy Shin](https://www.behance.net/adagio07), the original author of the used design.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Please kindly fix this broken link, the MIT License is a permissive software license that places few restrictions of reuse.  Perhaps you may need to add License file to your root folder and customize this [MIT template](https://choosealicense.com/licenses/mit/).